authors/apps/authentication/urls.py

```diff
@@ -1,11 +1,11 @@
 from django.conf.urls import url
-
+from django.urls import path
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
 urlpatterns = [
-    url(r'^user/?$', UserRetrieveUpdateAPIView.as_view()),
-    url(r'^users/?$', RegistrationAPIView.as_view()),
-    url(r'^users/login/?$', LoginAPIView.as_view()),
+    path('user/', UserRetrieveUpdateAPIView.as_view()),
```

Comment:
This is good work man, I dont really understand the rest so much since I havent gone through django so much but I recognise this bit right here